utils/Retail/Search/on_search.ts

```diff
@@ -487,7 +489,8 @@ export const checkOnsearch = (data: any, msgIdSet: any) => {
 
           if (item.quantity && item.quantity.maximum && typeof item.quantity.maximum.count === 'string') {
             const maxCount = parseInt(item.quantity.maximum.count, 10)
-            if (maxCount !== 99 && maxCount <= 0) {
+            const availCount = parseInt(item.quantity.available.count, 10)
+            if (availCount == 99 && maxCount <= 0) {
```

Comment:
max count can be 0 in case inventory is out of stock

Reply:
But in that case avail count will not be 99